controllers/topics.controller.js

```diff
@@ -0,0 +1,10 @@
+const { findTopics } = require("../models/topics.model");
+
+exports.getTopics = (req, res, next) => {
+  findTopics()
+    .then((result) => {
+      const topics = result.rows;
```

Comment:
As a general pattern I'd do the extracting of the rows in the model. This is logic to do with the result of a database call which is the models job. 

The controller should just get back the relevant data from the model.